website/src/pages/3million.tsx

```
@@ -0,0 +1,74 @@
+          content="Celebrate 3,000,000 downloads of Podman Desktop! Thank you to everyone who provides feedback and helps us improve."
+        />
+        <meta name="twitter:image" content="https://podman-desktop.io/img/3million-celebration.jpg" />
+      </Head>
+
+      <div
+        style={{
+          margin: 0,
+          padding: 0,
+          overflow: 'hidden',
+          minHeight: '100vh',
+        }}></div>
```

Biome (2.1.2)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Safe fix: Use a self-closing element instead.


(lint/style/useSelfClosingElements)